recipes/lz4/all/conanfile.py

```diff
@@ -1,12 +1,12 @@
 from conan import ConanFile
 from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
-from conan.tools.files import apply_conandata_patches, copy, get, rmdir, save
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, rmdir, save
 from conan.tools.microsoft import is_msvc
 from conan.tools.scm import Version
 import os
 import textwrap
 
-required_conan_version = ">=1.50.0"
+required_conan_version = ">=1.54.0"
```

Comment:
```suggestion
required_conan_version = ">=1.53.0"
```

Reply:
But if you want to remove `tc.cache_variables["CMAKE_POLICY_DEFAULT_CMP0077"] = "NEW", yes 1.54.0 would be required.

Reply:
Didn't think that far ahead for this PR :) As my knowledge is limited, I'm trying to change as little as I can, so I'll leave any proper changes to pros like you :)